client/my-sites/checkout/utils.ts

```diff
@@ -135,3 +135,24 @@ export function getAffiliateCouponLabel(): string {
 	// translators: The label of the coupon line item in checkout
 	return translate( 'Exclusive Offer Applied' );
 }
+
+export function getCouponLabel(
+	originalLabel: string,
+	experimentVariationName: string | null
+): string {
+	return experimentVariationName === 'treatment' ? translate( 'Offer Applied' ) : originalLabel;
+}
+
+export function isCouponBoxHidden(
```

Comment:
I think we can now remove this function and replace it inline with `'treatment' === experimentVariationName`, right? After the latest change, users will never be assigned to treatment if the cart contains a DIFM product.

Reply:
You are right, but I think it might be best to actually leave that code logic in. 
If the test validates our hypothesis and we proceed with the launch, we’ll need the DIFM code logic, as we’ll no longer be using the treatment value.